getting-started/prerequisites.md

```diff
@@ -21,11 +21,15 @@ $ go1.18.3 version
 go version go1.18.3 darwin/amd64
 ```
 
-You also need the following shared object files for your architecture and
-operating system from Nextmv. If not already installed, follow the instructions
-[here](https://docs.nextmv.io/cli/installation) to install and configure
-Nextmv's cli tool. Once you have successfully configured your api key, run
-`nextmv sdk get` to download the shared object files to `~/.nextmv/lib`. Add a
+You also need Nextmv's cli tool. _You must have access to a private beta for
+this feature. Contact [support][support] for more information_.
+
+If not already installed and configured, follow the instructions
+[here][cli-install]. The full reference for the Nextmv cli can be found
+[here][cli-reference].
+
+Once you have successfully configured your api key, run `nextmv sdk get` to
+download the necessary binary files to `~/.nextmv/lib`. Add a
```

Comment:
Does the user need to know where our binaries live? Just wondering. :shrug:

Reply:
I think you are right. They don't need to know where they are, `nextmv-cli` should take care of that for them.